src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -345,8 +345,8 @@ public function symlink($originDir, $targetDir, $copyOnWindows = false)
     /**
      * Given an existing path, convert it to a path relative to a given starting path.
      *
-     * @param string $endPath   Absolute path of target
-     * @param string $startPath Absolute path where traversal begins
+     * @param string $endPath   Path of target
+     * @param string $startPath Path where traversal begins
```

Comment:
This is a BC break. The method expects the paths to be absolute as documented.

Reply:
I opened #24202 to deprecate support for relative paths in Symfony 3.4

Reply:
@ausi looks like another later commit removed the "Absolute" again

Reply:
Yes, as i wrote in https://github.com/symfony/symfony/pull/22321#issuecomment-321475428

Reply:
Okay, I think we should revert that so that we do not advertise the support for relative paths, but only do that silently.

Reply:
Adding the “Absolute” back should then be part of #24202 I think. Otherwise I would have to remove the unit tests as @chalasr wrote in https://github.com/symfony/symfony/pull/22321#discussion_r132054398

Reply:
I see it different here: "Absolute" should still be part of the docblock to make our intent clear that we want this method to receive absolute paths. People who not look at the tests do not need to know that this currently works with relative paths too.

Reply:
@chalasr explicitly wrote:

> Tests should respect the contract which is to pass an absolute path.

Please discuss this with @chalasr first. I don’t want to change this pull request every two weeks. If you both agree, I can change the doccomment to whatever you want :)

Reply:
Just discussed with @xabbuh and we come to the conclusion that the docblock clearly states the passed path must be absolute, that has been decided when introducing the method and should not be changed now. 
Given we have a test in place for relative paths, let's improve the current implementation regarding relative paths as done here, so we can officially deprecate passing relative paths to the method in #24202 (or officially support any path, changing the docblock, but I don't think it's the right direction as currently discussed on the PR) and move forward.

Reply:
I restored the original doccomment in 193cdf802997e2b469e77d919cf74a740e2ba4ee now.
@chalasr @xabbuh is this pull request now ready to be approved?